extensions/skins/conversation_v1/Conversation.js

```diff
@@ -208,11 +208,22 @@ oppia.directive('conversationSkin', [function() {
       $scope.PANEL_TUTOR = 'tutor';
       $scope.PANEL_SUPPLEMENTAL = 'supplemental';
 
+      $scope.helpHtml = null;
+      $scope.continueToNextCardHtml = null;
```

Comment:
I find it confusing that both $scope.helpHtml and $scope.continueToNextCardHtml contain the content of the help card. It causes twice the amount of work down the road.

Instead, what do you think of having $scope.helpHtml always contain the content of the help card, but then having a flag like $scope.continueToNextCard = false which determines whether or not there should also be a continue button?